f/pabstMarkers/defines_unitsAndGroups.sqf

```diff
@@ -1,266 +1,372 @@
 #define UNIT_MARKERS [ \
```

Comment:
You reverted code conventions here and broke the naming convention for the marine unit names. Are we planning on adding a new set of units to the mission.sqm? The group name won't be seen on the map, it just shows A1/A2 or what ever, this will only effect spectate and orbat.